04-writing_a_DMS_plan.Rmd

```diff
@@ -5,20 +5,96 @@ ottrpal::set_knitr_image_path()
 
 # Writing a DMS Plan
 
+In this section we will guide you through the general requirements for the various sections of the DMS plan. 
+&nbsp;
+&nbsp;
 
- DMS elements
-   - less than 2 pages
-   - Initially not public now.. but may in the future (so don't include anything confidential or proprietary)
-    1. Data type
-    2. tools/code (note code is not required to be shared)
-    3. standards
-    4. data preservation, access, timelines
-      - no later than the time of publication of findings or the end of the award (whatever comes first) This probably stresses people out... maybe you can justify longer timelines? 
-      - renewals can have updated DMS plan with a different timeline, but if not approved must use previous DMS date
-      - no cost extension also allows for more time
-    5. access, distribution, reuse
-      - no specification of how long need to be available, journals may have other requirements
-    6. oversight - who will compliance be managed by
+General requirements:
+
+- The plan must be **less than 2 pages**.
+- Initially DMS plans will not be made public, but they **may be made public** in the future, so do not include anything that might be confidential or proprietary in your plan.
+
+These following are the major sections of the plan:
+
+1. **Data type** - describe what data (amount and type) will be generated over the course of funding and what data will or will not be shared
+2. **Tools, software, and code** (note code is not required to be shared) - describe in what tools you intend to use to manage and analyze the data
+3. **Standards** - describe any standards that you might use for your data
+4. **Data preservation, access, timelines** - describe where the data will be made available and when
+5. **Access, distribution, reuse considerations** - describe how you have carefully considered any reasons that might limit sharing
+6. **Oversight** - describe who will manage compliance of the DMS plan
+    
+    
+##  Data Type
+
+A. Types and amount data
+
+- Describe what kind of data will be generated over the course of funding and the expected amounts of data for each.
+
+B. Data Sharing
+
+- Describe what data from the project will shared or not shared and justify this decision.
+
+C. Metadata, other relevant data, and associated documentation
+
+- List information about metadata.
+- List other possibly relevant data (that you think is not considered scientific).
+- List any relevant information about how you plan to gather the data (ex. study protocols and data collection instruments) that you plan to share to make it easier for others to reproduce your work.
+
+
+## Related Tools, Software and/or Code:
+
+- Describe any software or code you intend to use to process or analyze the data (provide the names of the tools and how they can be accessed).
```

Comment:
Could we provide some examples ( I think we might need to get this from NIH) - e.g., R version 4.2.0 (link)

Reply:
I think it's fine to add this later, just something to think about